mantle/platform/api/packet/api.go

```diff
@@ -455,7 +463,7 @@ func waitForInstall(address string) (err error) {
 		conn, err = dialer.Dial("tcp", address+":9")
 		if err == nil {
 			defer conn.Close()
-			conn.SetDeadline(deadline)
+			_ = conn.SetDeadline(deadline)
```

Comment:
Why not leave as is for this line being that there is nothing being set for use or expected to be used?

Reply:
Thank you for review. It's tricky here. Since golang-ci lint requires to deal with all the errors, I think this error is not vital. So, just assign to blank in order to avoid the error message from golang-ci lint. Or we can return the error if it's not nil?

Reply:
I follow. What about using nolint? https://golangci-lint.run/usage/false-positives/

Reply:
Good idea. `nolint` seems much better than blank assignment.